Implemented import.py using new Weaviate Python Client

miniMaddy opened the pull request:
Issue: #1 

What changed: Implemented the demo using new Weaviate Python client.

@iamleonie, please review the PR, thank you!

iamleonie reviewed (changes requested):
Hey @miniMaddy, thanks for your contribution. I have a request for some changes.

We were [discussing on Slack](https://weaviate.slack.com/archives/C05TR90PMT4/p1696579221084489?thread_ts=1696570870.673199&cid=C05TR90PMT4) with another contributor how we should approach this best.

> The upcoming releases (hopefully 1.22) of Weaviate should be compatible with the new client (v4), as the gRPC back-end will be added before the client is final. I think using a preview image is and just updating the requirements is fine, and aim to swap it out with the official release once it is launched.

So, we'd prefer to replace the old client with the new client instead of having a separate folder. 

Thank you so much!

miniMaddy commented:
Issue #1 
Hello, @iamleonie , I have replaced the old client with the new client in the demo. I haven't made any significant changes made by @markgunnels for the readme and left it as it is.

Changes Made:
- docker-compose.yaml
- import.py
- deleted helper.py as we new client automatically generated uuid.

iamleonie commented:
Thank you so much for the changes. With the new Python client API changes, we will review the PR more in-depth, so it can take a little time to review your changes. We will get back to you shortly.

databyjp commented on import.py:
```diff
@@ -1,72 +1,57 @@
+print(client.collection.delete("Wine"))
+
+# Creating a new collection with the defined schema
+articles = client.collection.create(
```
This could probably be `reviews` or something like that imo.

databyjp commented on readme.md:
```diff
@@ -60,7 +60,7 @@ Before you can run the project, you need to have Docker, Docker Compose, and Pyt
 1. Start up Weaviate: `docker-compose up -d`. Once completed, Weaviate is running on [`http://localhost:8080`]().
 2. Run `python import.py` to import 2500 wines to Weaviate.
-3. Navigate to [console.semi.technology](https://console.semi.technology/), connect to `http://localhost:8080`, navigate to the query module, and happy querying!
+3. The data is now stored in the weaviate client. You can experiment with it using a python notebook or a python file.
```
Note that the `client` is the Python library, so it would not be accurate. Suggest saying it gets saved in the Weaviate `instance` or `database`.

```suggestion
3. The data is now stored in the Weaviate instance. You can experiment with it using a python notebook or a python file.
```

databyjp commented:
Looks good! I've made a couple of comments :)

miniMaddy commented:
I pushed the changes suggested by JP Hwang.
